### ISSUE TITLE
[apigateway] support SecurityPolicy attribute as part of the DomainName resource

### ISSUE THREAD

**Issue by markusl** [aws/aws-cdk#3862]
### What is the problem?

The following code will create a custom domain name with "Security Policy TLS 1.0" instead of TLS 1.2. Expected result would be the latest TLS version.

```javascript
    const api = new apigateway.LambdaRestApi(lambdaStack, `${env}-${componentName}-LambdaApis`, {
        handler: webhookLambda,
        proxy: false,
        deployOptions: {
            stageName: env,
        },
        domainName: {
            domainName: `webhook-${env}--`,
            certificate: cm.Certificate.fromCertificateArn(lambdaStack, `${env}Cert`, cert),
        },
    });
```

### Environment

  - **CDK CLI Version:** 1.6.0
  - **OS:** Macbook
  - **Language:** Typescript

**Comment by nmussy** [comment]
CloudFormation is currently lacking the ability to update the security policy and set the minimum TLS version. You can follow the progress in [aws-cloudformation-coverage-roadmap#3](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/3), it's apparently coming soon.

**Comment by nmussy** [comment]
Looks like the CloudFormation support was just added as of `7.3.0` 🙌

**Comment by kennu** [comment]
Came here to say the same thing :-) It's now live https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/3

**Comment by JackChap** [comment]
Does anyone know if this is in implemented CDK as of 1.21.0?
